src/Connector.sol

```diff
@@ -167,16 +159,14 @@ contract CentrifugeConnector {
         bytes16 trancheId,
         address user,
         uint256 amount,
-        string memory domainName
+        uint32 destinationDomain
```

Comment:
Are we ok with it being an arbirtrary `uint32` value? I thought we would also have an enum and restrict a `domain` argument to be one of that enum's variants?

Reply:
cc @offerijns

Reply:
> I thought we would also have an enum and restrict a domain argument to be one of that enum's variants?

Yeah this is what should happen